routes/api.php

```diff
@@ -1,11 +1,20 @@
 <?php
 
+use App\Http\Controllers\Api\PublisherController;
 use App\Http\Controllers\Api\UserController;
 use Illuminate\Support\Facades\Route;
 
 Route::post('/auth/login', [UserController::class, 'login']);
 
-Route::middleware('auth:sanctum')->group(function(){
+Route::middleware('auth:sanctum')->group(function () {
     Route::post('/auth/register', [UserController::class, 'register']);
     Route::post('/auth/logout', [UserController::class, 'logout']);
-});
\ No newline at end of file
+
+    // Publisher routes
+
+    Route::post('/auth/publishers/create', [PublisherController::class, 'createPublisher'])->name('publishers.create');
+
+    Route::get('/auth/publishers/{id}', [PublisherController::class, 'viewPublisher'])->middleware('checkLibrarian')->name('publishers.view');
```

Comment:
@radosevic-dr Which middleware is this?

Reply:
Same as in Route::get('/auth/users/{user}', [UserController::class, 'viewUser'])->middleware('checkLibrarian')->name('user.show'); checking for librarian but route not visible in this branch, not merged jet

Reply:
@radosevic-dr Got it